src/server/gc_worker/gc_worker.rs

```diff
@@ -740,20 +746,20 @@ where
         self.flow_info_sender
             .send(FlowInfo::BeforeUnsafeDestroyRange(ctx.region_id))
             .unwrap();
-        let local_storage = self.engine.kv_engine().unwrap();
 
-        // Convert keys to RocksDB layer form
-        // TODO: Logic coupled with raftstore's implementation. Maybe better design is
-        // to do it in somewhere of the same layer with apply_worker.
-        let start_data_key = keys::data_key(start_key.as_encoded());
-        let end_data_key = keys::data_end_key(end_key.as_encoded());
+        if let Some(local_storage) = self.engine.kv_engine() {
```

Comment:
please add comment this branch is for single rocksdb case.

Reply:
Fine.